lib/EnzymeCore/src/EnzymeCore.jl

```diff
@@ -257,4 +257,9 @@ function compiler_job_from_backend end
 
 include("rules.jl")
 
+if !isdefined(Base, :get_extension)
```

Comment:
@wsmoses the original PR by @devmotion had the backwards compatibility https://github.com/EnzymeAD/Enzyme.jl/pull/954/files#diff-94319485f88a6e9c6ce4669ce81ce9d3a46dab48d8c9e579b22272fc798a39a9R172